internal/model/plugin_test.go

```diff
@@ -156,6 +157,7 @@ func TestPluginsToWriter(t *testing.T) {
       "version": "2.0.0"
     },
     "enterprise": false,
+    "hosting": "",
```

Comment:
Does it make sense to have one test assertion here for the hosting being cloud or on-prem? As opposed to only having the empty string case.

Reply:
Makes sense. Added :+1: